### ISSUE TITLE
Ghost blank admin page

### ISSUE THREAD

**Issue by acroyear** [TryGhost/Ghost#6578]
### Issue Summary

The ghost admin page, and the sign up page, give me a blank screen and javascript errors on Chrome 48, OSX 10.11.

It works in Firefox 44.

### Steps to Reproduce

1. Just go to the ghost admin page on that platform.
2. View console, see the following:

Uncaught TypeError: RegExp.prototype.sticky getter called on non-RegExp object

Uncaught Error: Could not find module `ember-resolver` imported from `ghost/app`

both from vendor.min.js

Any other info e.g. Why do you consider this to be a bug? What did you expect to happen instead?

A working admin page.

### Technical details:

* Ghost Version: 0.7.4 and 0.7.6
* Node Version: unknown
* Browser/OS: Chrome/48.0.2564.116 OSX 10.11.3
* Database: mysql

I'll note: this is not the only page that has broken with Chrome 48 in my experiences so far (even Amazon has pages that stopped working).

**Comment by acroyear** [comment]
I use ghost in a hosted environment. I have very little control over the node version or other such matters. Scriptaculous does the install and upgrade.

However, the regex thing to me seems the key...

and looking that up, I just saw this may be a dup of https://github.com/TryGhost/Ghost/issues/6102

So feel free to close, and I'll await a Chrome update and rely on Firefox for the time being. I can't disable "experimental Javascript" as there is another ES6 feature I need for other work that would likely be removed if I did that.
